Configuration/DataProcessing/test/cosmicsEra_Run2_2018_HI_t.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
```

Comment:
IIRC, these test files are not really used.
So, this file is not really needed.